Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -271,6 +271,12 @@ class PlaceDetailsCardView: UIView {
         }
     }
 
+    func showEvent(atPlace place: Place) {
+        updateEventUI(forPlace: place)
+        self.setNeedsLayout()
+        self.layoutIfNeeded()
```

Comment:
I think these should be inside updateEventUI.

In that case, why not just expose `updateEventUI` and get rid of the callthrough?